sdk/tests/conformance2/rendering/blitframebuffer-filter-outofbounds.html

```diff
@@ -111,6 +115,10 @@
             [4, 4, 8, 8],
             [8, 8, 8, 8],
             [6, 6, 8, 8],
+            [5, 5, 8, 8],
```

Comment:
And I carefully designed another test, it is the next one in this pr: blitting from [-2, -2, 4, 4] to [3, 3, 10, 10]. 
The scaleFactor = 7/6. 
The untouched src region width/height = 2. The untouched dst region would be 2 * 7/6 = 7/3. 
So the effective dst region [3+7/3, 10] = [5.333333, 10]. Then the pixel (5, 5)'s center (5.5, 5.5) lies inside the effective dst region, so the blitted area (intersects with draw buffer) would be [5, 5, 8, 8].